test/cets_SUITE.erl

```diff
@@ -2471,6 +2473,19 @@ disco_node_start_timestamp_is_updated_after_node_restarts(Config) ->
     simulate_disco_restart(Setup),
     wait_for_disco_timestamp_to_be_updated(Disco, node_start_timestamps, Node2, OldTimestamp).
 
+disco_pang_result_arrives_after_nodeup(Config) ->
```

Comment:
Please make the test case name consistent with what it does. It says that `pang` arrives after `nodeup`, but instead we just always return `pang`. Also, the test could be misleading, because it looks like we expect everything to be OK despite ping result always being `pang`.